cmd/common-main.go

```diff
@@ -262,10 +262,14 @@ func handleCommonEnvVars() {
 		logger.FatalIf(err, "Unable to initialize etcd with %s", etcdEndpoints)
 	}
 
-	globalDomainName, globalIsEnvDomainName = os.LookupEnv("MINIO_DOMAIN")
-	if globalDomainName != "" {
-		if _, ok = dns2.IsDomainName(globalDomainName); !ok {
-			logger.Fatal(uiErrInvalidDomainValue(nil).Msg("Unknown value `%s`", globalDomainName), "Invalid MINIO_DOMAIN value in environment variable")
+	v, ok := os.LookupEnv("MINIO_DOMAIN")
```

Comment:
this should be MINIO_DOMAINS for consistency across envs

Reply:
There is a good reason it's not DOMAINS as it's not multiple domains simultaneously accessing Minio it is always a DOMAIN. The point is that they all are just DOMAIN for Minio at the request level. It is similar to `--encrypt-key` option we don't say it `--encrypt-keys` or `--encrypts` for `--encrypt` option.

An additional benefit is avoiding breakage in backward compatibility, most users won't need this feature anyways.

Reply:
that's a good argument